src/aiida/orm/utils/node.py

```diff
@@ -60,8 +60,8 @@ def load_node_class(type_string):
             return Data
 
     if base_path.startswith('process'):
-        entry_point_name = base_path.removeprefix('data.')
-        return load_entry_point('aiida.node', entry_point_name)
+        entry_point_name = base_path.removeprefix('nodes.')
+        return load_entry_point('aiida.node', entry_point_name) 
```

Comment:
I believe the original code was incorrect

```suggestion
        entry_point_name = base_path.removeprefix('process.')
        return load_entry_point('aiida.node', entry_point_name) 
```